Index: cinnamon-control-center/panels/screen/cc-screen-panel.c
===================================================================
--- cinnamon-control-center.orig/panels/screen/cc-screen-panel.c	2014-02-20 14:53:51.720885523 +0100
+++ cinnamon-control-center/panels/screen/cc-screen-panel.c	2014-02-20 14:53:51.716885687 +0100
@@ -446,8 +446,7 @@
       gtk_tree_model_get (model, &iter,
                           1, &value_tmp,
                           -1);
-      if (value == value_tmp ||
-          (value_tmp > value_prev && value < value_tmp))
+      if (value == value_tmp)
         {
           gtk_combo_box_set_active_iter (combo_box, &iter);
           return;
Index: cinnamon-control-center/panels/screen/screen.ui
===================================================================
--- cinnamon-control-center.orig/panels/screen/screen.ui	2014-02-20 14:53:51.720885523 +0100
+++ cinnamon-control-center/panels/screen/screen.ui	2014-02-20 14:53:51.716885687 +0100
@@ -45,6 +45,10 @@
         <col id="0" translatable="yes">1 hour</col>
         <col id="1">3600</col>
       </row>
+      <row>
+	<col id="0" translatable="yes">Never</col>
+	<col id="1">0</col>
+      </row>
     </data>
   </object>
   <object class="GtkListStore" id="screen_brightness_liststore">
