Description: Cleaned up build files
Author: monsta
Origin: PR #91 for upstream - https://github.com/linuxmint/cinnamon-control-center/pull/91
--- a/Makefile.am
+++ b/Makefile.am
@@ -26,7 +26,7 @@
 EXTRA_DIST = \
 	MAINTAINERS
 
-DISTCHECK_CONFIGURE_FLAGS = --disable-update-mimedb --enable-gtk-doc --disable-maintainer-mode
+DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc --disable-maintainer-mode
 
 GITIGNOREFILES=m4
 
--- a/configure.ac
+++ b/configure.ac
@@ -97,7 +97,6 @@
 MODEM_MANAGER_REQUIRED_VERSION=0.7
 LIBNOTIFY_REQUIRED_VERSION=0.7.3
 CINNAMON_DESKTOP_REQUIRED_VERSION=1.0.0
-CLUTTER_REQUIRED_VERSION=1.11.3
 CSD_REQUIRED_VERSION=1.0.0
 LIBWACOM_REQUIRED_VERSION=0.7
 
@@ -172,7 +171,7 @@
 
 AC_ARG_ENABLE([cups],
               AS_HELP_STRING([--disable-cups], [disable CUPS support (default: enabled)]),,
-              [enable_cups=yes])
+              [enable_cups=no])
 
 if test x"$enable_cups" != x"no" ; then
   AC_PROG_SED
@@ -263,17 +262,6 @@
 AM_CONDITIONAL(WITH_LIBSOCIALWEB, test "x$with_libsocialweb" = "xyes")
 
 
-dnl =======================================
-dnl Update Mime Database
-dnl =======================================
-
-AC_PATH_PROG(UPDATE_MIME_DATABASE, update-mime-database, no)
-
-AC_ARG_ENABLE(update-mimedb,
-   AS_HELP_STRING([--disable-update-mimedb],
-                  [do not update mime database after installation]),,
-                   enable_update_mimedb=yes)
-AM_CONDITIONAL(ENABLE_UPDATE_MIMEDB, test x$enable_update_mimedb = xyes)
 
 CONTROL_CENTER_VERSION=cinnamon_control_center_version
 AC_SUBST(CONTROL_CENTER_VERSION)
@@ -426,22 +414,17 @@
 else
 	AC_MSG_NOTICE([   Network panel disabled])
 fi
-if test "x$with_libsocialweb" = "xyes"; then
-	AC_MSG_NOTICE([** libsocialweb (Background panel Flickr support)])
+
+if test "x$build_wacom" = "xtrue"; then
+	AC_MSG_NOTICE([** Wacom support (Wacom panel)])
 else
-	AC_MSG_NOTICE([   Background panel Flickr support disabled])
+	AC_MSG_NOTICE([ Wacom support disabled])
 fi
+
 if test "x$with_systemd" = "xyes"; then
 	AC_MSG_NOTICE([** systemd (Systemd session tracking)])
 else
 	AC_MSG_NOTICE([   Using ConsoleKit for session tracking])
 fi
 
-AC_MSG_NOTICE([   Wacom support:  $build_wacom])
-
-if test "x$enable_ibus" == "xyes"; then
-	AC_MSG_NOTICE([** IBus (Region panel IBus support)])
-else
-	AC_MSG_NOTICE([   Region panel IBus support disabled])
-fi
 AC_MSG_NOTICE([End options])
